pypdf/generic/_annotations.py

```diff
@@ -174,7 +178,7 @@ def popup(
                 NameObject("/Subtype"): NameObject("/Popup"),
                 NameObject("/Rect"): RectangleObject(rect),
                 NameObject("/Open"): BooleanObject(open),
-                NameObject("/Flags"): NumberObject(flags),
+                NameObject("/F"): NumberObject(flags),
```

Comment:
Was that a bug/typo before?

Reply:
Good guess! it is a typo error! thanks for detecting it